lib/http.js

```diff
@@ -13,10 +13,10 @@ function httpRequest(sdk, options) {
   var url = options.url,
       method = options.method,
       args = options.args,
-      cacheState = options.cacheState,
+      cacheAuthnState = options.cacheAuthnState,
```

Comment:
A bit pedantic, but how about "saveAuthnState"? The difference for me is that we're using "cacheResponse" to mean "let's save this response for up to the DEFAULT_CACHE_DURATION, and I do not expect the response to change very frequently", where authState gets updated on every request and is expected to hold different values as the user goes through the auth flow